conan/cli/commands/install.py

```diff
@@ -33,6 +33,8 @@ def install(conan_api, parser, *args):
                         help='Deploy using the provided deployer to the output folder')
     parser.add_argument("--deployer-folder",
                         help="Deployer output folder, base build folder by default if not set")
+    parser.add_argument("--deployer-package", action="append",
+                        help="Execute the deployment of the specific packages")
```

Comment:
Not sure if this is too late to amend, but I think that this is an ambiguous description
```suggestion
                        help="Execute the deploy() method of the named package references")
```

Reply:
Agree, lets change it (in a new PR)

Reply:
Fixed